client/adjudicate.go

```diff
@@ -447,7 +447,9 @@ func (c *Channel) subChannelStateMap() (states channel.StateMap, err error) {
 // ensureRegistered ensures that the channel is registered.
 func (c *Channel) ensureRegistered(ctx context.Context) error {
 	phase := c.Phase()
-	if phase == channel.Registered || phase == channel.Progressed {
+	if phase == channel.Registered ||
+		phase == channel.Progressing ||
```

Comment:
Why do we also treat a channel as registered if it's currently still progressing?

Reply:
Answered in a meeting.